content/pages/1.install.md

```diff
@@ -39,15 +43,21 @@ The easiest way install ``mycli`` on a Mac is to use [homebrew].
     $ sudo apt-get install mycli
 ```
 
-### RHEL, Centos, Fedora:
+### Fedora
+
+```
+    $ sudo dnf install mycli
+```
+
+### RHEL, Centos:
 
-I haven't built an RPM package for mycli yet. So please use `pip` to install `mycli`. You can install pip on your system using:
+We don't have an RPM package for mycli, yet. So, please use `pip` to install `mycli`. You can install pip on your system using:
```

Comment:
The RPM can be used on Centos/RHEL but most of the dependencies are missing. I've created a repository with all the dependencies on https://copr.fedorainfracloud.org/coprs/meeuw/mycli-epel/

I'd like to get this into Fedora-EPEL but haven't found the time to do this.